shell/initialize/app-extended.js

```diff
@@ -70,19 +70,18 @@ async function extendApp(vueApp) {
     req:     undefined,
     res:     undefined
   });
-
   await installInjectedPlugins(appPartials, vueApp);
 
   // Wait for async component to be resolved first
   await new Promise((resolve, reject) => {
     // Ignore 404s rather than blindly replacing URL in browser
-    const { route } = router.resolve(appPartials.context.route.fullPath);
+    const route = router.resolve(router.currentRoute.value.fullPath);
 
     if (!route.matched.length) {
       return resolve();
     }
 
-    router.replace(appPartials.context.route.fullPath, resolve, (err) => {
+    router.replace(router.currentRoute.value.fullPath, resolve, (err) => {
```

Comment:
I don't know if it matters, but this can be a candidate for `master`. Potentially one less change to track when we make the break for Vue3.

Reply:
I'd certainly be willing to try and see how e2e tests deal with it.